api/src/pcapi/routes/backoffice/offers/forms.py

```diff
@@ -251,6 +255,15 @@ def __init__(self, *args: list, **kwargs: dict):
         search_inline=True,
         field_list_compatibility=True,
     )
+    address = fields.PCTomSelectField(
+        "Adresses (autocomplétion expérimentale)",
```

Comment:
Backoffice v4 bêta 😮

Reply:
Je dis expérimental, car ce n'est pas parfait, en fait on récupère un nombre d'adresses de l'API Adresse, mais potentiellement aucune dans notre base de données, alors qu'on aurait pu avoir des propositions en itérant sur plus de résultats de l'API Adresse. Je ne pense pas que ce soit utile cependant, car on visera plutôt une adresse préciser.

En fait ce champ de recherche a pour principal intérêt de pouvoir le remplir dans le liens des offres depuis chaque ligne de la liste des adresses.